wordpress-mobile/WordPress-Android
# Implement blockSite action on discover Reader tab

Parent issue #12028

Adds "Block site" action to the items on the discover tab in Reader. 

To test:
1. Enable RI FF
2. Open discover Reader tab
3. Click on the more button on one of the items
4. Select "Block Site"
5. Notice all posts from this site disappear
6. Snackbar with "Undo action" is shown
7. Click on "Undo" and notice the posts re-appear

-------
1. Open discover Reader tab
1. Turn on airplane mode
2. Click on the more button on one of the items
4. Select "Block Site"
5. Notice an error is shown

PR submission checklist:
